src/Sensorario/DiamondKata/DiamondKata.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace Sensorario\DiamondKata;
+
+final class DiamondKata
+{
+    const EOL = "\n";
+
+    private $params;
+
+    private function __construct(array $params)
+    {
+        $this->params['input'] = $params['input'];
+    }
+
+    public static function createFrom($input)
+    {
+        return new self([
+            'input' => $input
+        ]);
+    }
+
+    public function output()
+    {
+        for ($i = 0; $i <= $this->size(); $i++) {
+            $base = str_pad(" ", ($i*2)+1);
+            $base[0] = chr(65 + $i);
+            $base[strlen($base)-1] = chr(65 + $i);
+
+            $lines[$i] = str_pad(
+                $base,
+                $this->size(),
+                " ",
+                STR_PAD_BOTH
+            ) . "\n";
+        }
+
```

Comment:
This method is a little bit unreadable :))) Maybe some variable extractions with possibly significant names could help us.. Ex: what ($i*2)+1 is? What chr(65+$i) is? What about strlen($base)-1 ? And so on ..